src/energyplus/ForwardTranslator/ForwardTranslateSizingZone.cpp

```diff
@@ -245,30 +243,15 @@ namespace energyplus {
     // This would be done in forwardTranslateControllerMechanicalVentilation except doing it here maintains proper order of the idf file.
 
     // Now that Multiple AirLoopHVACs serving the same zone are possible, need to loop on all
+    // NOTE: translateControllerMechnicalVentilation ensures ControllerMechanicalVentilation doesn't end up with no extensible groups!
     for (const auto& airLoopHVAC : thermalZone.airLoopHVACs()) {
       if (boost::optional<model::AirLoopHVACOutdoorAirSystem> oaSystem = airLoopHVAC.airLoopHVACOutdoorAirSystem()) {
         model::ControllerOutdoorAir controllerOutdoorAir = oaSystem->getControllerOutdoorAir();
         model::ControllerMechanicalVentilation controllerMechanicalVentilation = controllerOutdoorAir.controllerMechanicalVentilation();
         if (boost::optional<IdfObject> _controllerMechanicalVentilation = translateAndMapModelObject(controllerMechanicalVentilation)) {
-          IdfExtensibleGroup eg = _controllerMechanicalVentilation->pushExtensibleGroup();
-
-          // Thermal Zone Name
-          eg.setString(Controller_MechanicalVentilationExtensibleFields::ZoneorZoneListName, name);
-
-          // DesignSpecificationOutdoorAir
-          std::vector<model::Space> spaces = thermalZone.spaces();
-
-          if (!spaces.empty()) {
-            if (boost::optional<model::DesignSpecificationOutdoorAir> designOASpec = spaces.front().designSpecificationOutdoorAir()) {
-              if (boost::optional<IdfObject> _designOASpec = translateAndMapModelObject(designOASpec.get())) {
-                eg.setString(Controller_MechanicalVentilationExtensibleFields::DesignSpecificationOutdoorAirObjectName, _designOASpec->name().get());
-              }
-            }
-          }
-
-          // DesignSpecificationZoneAirDistributionObjectName
-          if (isDSZADTranslated) {
-            eg.setString(Controller_MechanicalVentilationExtensibleFields::DesignSpecificationZoneAirDistributionObjectName, dSZADName);
+          if (auto dsoaOrList_ = getOrCreateThermalZoneDSOA(thermalZone)) {
+            IdfExtensibleGroup eg =
+              _controllerMechanicalVentilation->pushExtensibleGroup({name, dsoaOrList_->nameString(), isDSZADTranslated ? dSZADName : ""});
```

Comment:
This is where the ControllerMV extensible fields are filled, and if it wasn't translated, nothing is done.